package.json

```diff
@@ -14,8 +14,16 @@
     "build": "turbo run build",
     "ci:test": "./scripts/ci-test.sh",
     "pretest": "turbo run pretest",
-    "test": "jest --no-cache --runInBand --coverage",
-    "test:watch": "jest --no-cache --watchAll",
+    "test": "jest --no-cache --runInBand --config './jest.all.config.ts'",
```

Comment:
When I run `pnpm ci:test` the `.browser` test files are ignored. Also, all `.node.browser` test files are running only on the `node` environment. Is this the expected behavior? Are we going to run all our tests on both environments? Or do we aim to only run some specific tests on the browser?

Running `pnpm test packages/{packageName}` runs the `.browser` tests but the `.node.browser` tests seem to be running only on the  `node` environment.

Reply:
These are good points and probably require some additional description _somewhere_. I'll definitely add to the PR description but maybe even in the contributing section of docs. But potentially the commands need a rethink.

1. `pnpm ci:test` default behaviour will only run the `node` tests. It can also run the `browser` tests with `pnpm ci:test --browser` as seen in the altered GitHub action:
https://github.com/FuelLabs/fuels-ts/blob/4a4bda6698cd9279a8479576234b305d6620dfe9/.github/workflows/test.yaml#L45
2. So the file path suffix allows it to be picked up by a config. This way a test can run on multiple environments. I believe there are many tests that we want to run across environments, therefore have suffixed them with `...node.browser.test.ts`. Anything that we only require on browser, should just be `...browser.test.ts`, similarly with `...node.test.ts`.
3. `pnpm test` is currently using `jest.all.config.ts' configuration which should be running all tests regardless of environment suffix, and run them using `node`. Which is the previous behaviour of `pnpm test`. However if it is not picking up certain tests, I will investigate that.

Reply:
I think `test:ci` should default to all tests, like the `pnpm test` command.

Then, we could have the option to run only a group in isolation.

```sh
pnpm ci:test # run all tests
pnpm ci:test --browser # only browser tests
pnpm ci:test --node # only node tests
```

Reply:
@arboleya @Torres-ssf Would you expect `pnpm ci:test` to run all tests as node, or all the tests in both environments, even if a test requires both environments?